lib/data/rates/energy_grants_calculator.yml

```diff
@@ -0,0 +1,3 @@
+- start_date: 2015-04-06
+  end_date: 3000-01-01
```

Comment:
Given that we know the the winter_fuel_payment_threshold changes each year, and that 1953-05-05 is valid for winter 2016/17, I think it might be better to use a start and end date of 2016-04-06 and 2017-04-05.

The behaviour of `RatesQuery#rates` means that this rate will continue to be used after 5 Apr 2017 until we add the threshold date for 2017/18.